ui/apps/ui/src/app/collections/data/all/adapter.data.ts

```diff
@@ -64,8 +64,18 @@ export const allCollectionsAdapter: IAdapter = {
       },
       {
         label: 'DOI',
-        value: data?.url || [],
-        filter: 'url',
+        value: data?.doi || [],
```

Comment:
@agpul `DOI` occurs only for Openaire-type resources. What do we want to do with trainings and services?
Do we want to present nothing or a `URL`?

Reply:
nothing:)

Reply:
ok so it's good as it is